### DISCUSSION TITLE
sionna-0.12 vs tf-nightly?

### DISCUSSION THREAD

**Discussion by nbecker** [NVlabs/sionna#89] [General]
Trying to use python-3.11.  tf-nightly seems to install on it.  Probably untested, but just thought I'd try it anyway.  But
```
 pip install sionna
...
The conflict is caused by:
    sionna 0.12.0 depends on tensorflow!=2.8.0, !=2.8.1, !=2.8.2, !=2.8.3, !=2.9.0, !=2.9.1 and >=2.7.4; sys_platform != "darwin"
```

```
>>> tf.version.VERSION
'2.12.0-dev20221228'
```

OK, isn't 2.12.0-dev20221228 >= 2.7.4?

**Comment by gmarcusm** [comment]
there are separate packages in PIP for Tensorflow stable `tensorflow` and Tensorflow nightly `tf-nightly`. In order to test the nightly packages you need to update the requirements accordingly.
